docs/av-stack-sync.1.md

```diff
@@ -0,0 +1,61 @@
+# av-stack-sync 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-sync - Synchronize stacked branches
+
+# SYNOPSIS
+
+`` av stack sync [--all | --current] [--no-push] [--no-fetch] [--prune]
+              [--trunk] [--continue | --abort | --skip] [--parent=<parent>]
+``
+
+# DESCRIPTION
+
+Synchronize stacked branches to be up-to-date with their parent branches.
+
+By default, this command will sync all branches starting at the root of the
+stack and recursively rebasing each branch based on the latest commit from the
+parent branch.
+
+If the --current flag is given, this command will not recursively sync dependent
+branches of the current branch within the stack. This allows you to make changes
+to the current branch before syncing the rest of the stack.
+
+If the --trunk flag is given, this command will synchronize changes from the
```

Comment:
I wonder if it's worth also flagging this in the description on line 17.
```
By default, this command does not synchronize changes from the mainline branch of the repository.
```

Reply:
Mentioned.